test/test_torch.py

```diff
@@ -5940,6 +5940,32 @@ def test_isinf_type(self, device):
         with self.assertRaises(TypeError):
             torch.isinf(1)  # Parameter must be a tensor
 
+    @unittest.skipIf(not TEST_NUMPY, 'NumPy not found')
```

Comment:
PyTorch now requires NumPy to run the test suite, so we no longer need to test for it. This line can be removed.

Reply:
Removed!